source/IntegrationTests/Migration/MeasurementsJsonToEbixStreamWriterTests.cs

```diff
@@ -28,54 +33,23 @@ public class MeasurementsJsonToEbixStreamWriterTests
     public async Task Given_ValidJson_When_WriteStreamAsync_Then_ReturnsValidMarketDocumentStream()
     {
         // Arrange
+        var serializer = new Serializer();
         var timeSeriesJsonToEbixStreamWriter = new MeasurementsJsonToEbixStreamWriter(
-            new Serializer(),
-            new JsonSerializerOptions { PropertyNameCaseInsensitive = true },
-            new MeasurementsToMarketActivityRecordTransformer());
+            serializer, new List<IDocumentWriter> { new MeteredDataForMeteringPointEbixDocumentWriter(new MessageRecordParser(serializer)) });
+        var meteredDataForMeteringPointEbixMessageParser = new MeteredDataForMeteringPointEbixMessageParser(
+            new EbixSchemaProvider(),
+            new Logger<MeteredDataForMeteringPointEbixMessageParser>(new LoggerFactory()));
 
         // Act
-        var stream = await timeSeriesJsonToEbixStreamWriter.WriteStreamAsync(JsonPayloadConstants.SingleTimeSeriesWithSingleObservation);
+        var timeSeriesPayload = Encoding.UTF8.GetBytes(JsonPayloadConstants.SingleTimeSeriesWithSingleObservation);
+        var stream = await timeSeriesJsonToEbixStreamWriter.WriteStreamAsync(new BinaryData(timeSeriesPayload));
```

Comment:
```suggestion
        var output = await timeSeriesJsonToEbixStreamWriter.WriteStreamAsync(new BinaryData(timeSeriesPayload));
```
`var stream` was confusing for me, since `item1` is the actual `stream`